Zara Antonia-Maria/L4/3.7/main.c

```diff
@@ -0,0 +1,34 @@
+///hexadecimal number to decimal
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main()
+{
+    char nr[1001];
+    int length, decNr=0, power=1,i;
+    scanf("%s",nr);
+    length=strlen(nr);
+    for(i=length-1;i>=0;i--)
+    {
+        if(nr[i]=='0')decNr=decNr+0*power;
```

Comment:
When you see so much code that gets repeated, you might want to think about using something else instead.
In this case, the best option would be to use a structure, but we're a few labs away from learning that.
Hold the thought, you'll see what I mean when we get there